system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/EndToEndTransferPostgresqlDefaultTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.test.e2e;
+
+import org.eclipse.edc.junit.annotations.PostgresqlDbIntegrationTest;
+import org.eclipse.edc.junit.extensions.EdcRuntimeExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.HashMap;
+
+import static org.eclipse.edc.test.e2e.PostgresUtil.createDatabase;
+
+@PostgresqlDbIntegrationTest
+class EndToEndTransferPostgresqlDefaultTest extends AbstractEndToEndTransfer {
```

Comment:
I'm not sure about the need to have an e2e tests that verifies the default datasource, I would just change the `EndToEndTransferPostgresqlTest` to make use of the default datasource.